src/Volume.ts

```diff
@@ -247,11 +248,12 @@ export default class Volume {
     if (noReload && !this.loadSpec.subregion.equals(this.loadSpecRequired.subregion)) {
       const currentScale = this.imageInfo.multiscaleLevel;
       // `LoadSpec.multiscaleLevel`, if specified, forces a cap on the scale level we can load.
-      const minScale = this.loadSpec.multiscaleLevel ?? 0;
+      const minLevel = this.loadSpec.multiscaleLevel ?? 0;
       // Loaders should cache loaded dimensions so that this call blocks no more than once per valid `LoadSpec`.
       const dims = await this.loader?.loadDims(this.loadSpecRequired);
-      if (dims && currentScale > minScale && dims[currentScale - 1].canLoad) {
-        noReload = false;
+      if (dims) {
+        const loadableLevel = estimateLevelForAtlas(dims.map(({ shape }) => [shape[2], shape[3], shape[4]]));
+        noReload = !dims || currentScale <= Math.max(loadableLevel, minLevel);
```

Comment:
`dims` is guaranteed to be truthy here already, so `!dims` always evaluates to false...
```suggestion
        noReload = currentScale <= Math.max(loadableLevel, minLevel);
```